index.html

```diff
@@ -14,6 +14,68 @@
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+    <header class="header-container">
+        <img src="./img/first-background.jpg" alt="">
+        <nav class="nav-bar">
+            <img class="karma-logo" src="./img/karma-logo.svg" alt="karma-logo">
+            
+                    <ul>
+                        <li> <a href="#"> Meet Karma </a></li>
+                        <li><a href="#"> How it works </a></li>
+                        <li><a href="#"> Store </a></li>
+                        <li><a href="#"> Blog </a></li>
+                        <li><a href="#"> Help </a></li>
+                        <li><a href="#"> Login </a></li>
+                    </ul>
+                </nav>
+
+        <div class="introducing-karma">
+        <h1>Introducing Karma</h1>
+        <p>Bring WiFi with you, everywhere you go.</p>
+        <div class="learn-more"> <a href="#">Learn more</a> </div>
+        </div>
+    </header>
+
+    <main>
+        <div class="main-container">
+            <p>Everyone needs a little Karma</p>
+            <div class="image-container">
+                <div class="image-device">
+                    <img src="./img/icon-devices.svg" alt="Internet for all devices">
+                    <div class="desc"><p>Internet for all devices</p></div>
+                </div>
+                    
+                <div class="image-coffee">
+                    <img src="./img/icon-coffee.svg" alt="icon-coffee">
```

Comment:
```suggestion
                    <img src="./img/icon-coffee.svg" alt="Coffee cup">
```
`alt` is a description of the image, so we can describe this one like "coffee cup". Same goes for the other images in this file